### PR TITLE
New navigation for 1.10

### PR DESCRIPTION
**Pull request by moxiegirl**
This PR is part of work across all projects to create product-specific menus to replace the current taxonomy. This menu is for Engine.

image
- Fixing the links
- Creating Engine specific menu

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

File: docs/admin/ambassador_pattern_linking.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
 <!--[metadata]>
 +++
+aliases = ["/articles/ambassador_pattern_linking/"]
```
Comment (thaJeztah) [inline docs/admin/ambassador_pattern_linking.md]: should the aliases start with `/engine/` ? Given that the current URL is https://docs.docker.com/engine/articles/ambassador_pattern_linking/

**Comment by thaJeztah** [comment]
Thanks @moxiegirl, some first impressions (overall, I like this :+1:);

- I think we should rename "Reference -> Docker Engine Commands" to "Command Line reference", and the "Reference -> Reference" menu to "API reference"
- I wonder if we should move the "docker.io accounts API" underneath "API reference"
- There's still the "Remote API clients" page, which is currently under the "old" Docker Reference menu

**Comment by moxiegirl** [comment]
@thaJeztah thanks for the review. Done.  The broken doc build is unrelated to this change.

@SvenDowideit can I get a review from you as well please.

**Comment by thaJeztah** [comment]
@moxiegirl Looked once more;

- Should we have "Quickstart containers" and "Understand the architecture" underneath "userguide"?
- If feels as if "Dockerize an application" and "User guide -> Learn by example" are overlapping quite a bit;
  I think I'd expected the "Dockerize an application" underneath the user guide.

Looks like you didn't fix the aliases yet; they should all have an `/engine/` prefix, because this one is broken for me;

http://192.168.99.100:8000/engine/articles/ambassador_pattern_linking/

**Comment by MHBauer** [comment]
Cool. How much of this is movement and renaming vs new content?

File: docs/installation/cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+parent="engine_install"
+weight="-60"
++++
+<![end-metadata]-->
+
+# Install Engine in the cloud
+
+* [Understand cloud install options and choose one](cloud.md)
+* [Example: Use Docker Machine to provision cloud hosts](cloud-ex-machine-ocean.md)
+* [Example: Manual install on a cloud provider](cloud-ex-aws.md)
```
Comment (thaJeztah) [inline docs/installation/cloud/index.md]: shouldn't we just rename `cloud/cloud.md` to `cloud/index.md`, or would that require the foldable menu-items to be clickable?

**Comment by moxiegirl** [comment]
@thaJeztah It would require the menu labels (foldable items) to be clickable.  We don't have that in this theme. Next one will have it.

@MHBauer I did tweak content slightly.  In much of docker/docker we were using generic Docker to mean Docker Engine.  I think the distinction was important.

Updated the aliases.

**Comment by thaJeztah** [comment]
@moxiegirl looks like the new "couchbase" document needs to be moved

screen shot 2016-01-26 at 09 47 20

**Comment by thaJeztah** [comment]
Other than that, LGTM (but we may need another check once all the other docs have been migrated)

**Comment by moxiegirl** [comment]
@thaJeztah Ready to go

**Comment by thaJeztah** [comment]
LGTM, thanks!

(note the menu may still be "off" if other projects still use the old structure; this will be fixed once all projects are migrated)

**Pull request merged**